source/tryGoal/tryGoal_service.js

```diff
@@ -15,6 +15,50 @@ services.factory('ConfigAPI', ['$resource', function($resource) {
     });
 }]);
 
+services.factory('agentHttpInterceptor', ['agentMemory', '$rootScope', function(agentMemory, $rootScope) {
+    
+    $rootScope.apiOutput = new Array();
+
+    return {
+
+        'request': function(config) {
+            
+            if (agentMemory.tryGoal && !config.url.endsWith('.html') && 
+            (config.url.endsWith('query') || config.url.endsWith('response'))) {
```

Comment:
should it be config.url.endsWith('response') or config.url.endsWith('request')?

Reply:
No, these are the query and response API endpoints rather than the type of requests (request or response).

Reply:
The log shows the start, query and response endpoint requests and responses at this stage. I'll add a note to the Jira ticket as such. This means for example the evidence tree API info isn't listed although as it's in another browser window it requires further thought on how to retrieve it. Again I'll comment in Jira ticket.